packages/shared/src/contexts/BootProvider.tsx

```diff
@@ -181,6 +181,7 @@ export const BootDataProvider = ({
       deviceId={deviceId}
       experimentation={cachedBootData?.exp}
       updateExperimentation={updateExperimentation}
+      isFetched={isFetched}
```

Comment:
We also have another state in this component named `initialLoad`, it might be better to combine checking if it is not null or undefined with `isFetched` since we watch for the changes of boot remote data and then set the result to state (hopefully we can do the boot refactoring soon) to understand we have now the value from the server. Just worried about a race condition.

Reply:
You might indeed be able to use this initialLoad for it, should result in same effect from what I can see.

Reply:
There could be a difference of 1 render. Once the query is done fetching, the `isFetched` is true now, but the `useEffect` to set the cachedBootData (since we set it to a new value in the `useEffect` has not been run yet and since the property value is now true, it would proceed with the previous cache data.

Reply:
I'll take a look at `initialLoad`.

It already runs with the cached data on the experiment, hence why you see the hijack page on the extension after logging in. You first see the feed, then get thrown back to the hijack page once `updateExperimentation` has been run and set it to the old cache. After I page reload it is all good and dandy.